docs/content/en/docs/reference/clusterspec/gitops.md

```diff
@@ -8,7 +8,133 @@ description: >
 
 # GitOps Support (Optional)
 EKS Anywhere can create clusters that supports GitOps configuration management with Flux. 
-In order to add GitOps support, you need to configure your cluster by updating the configuration file before creating the cluster. 
+In order to add GitOps support, you need to configure your cluster by updating the configuration file before creating the cluster.
+We currently support two types of configurations: `FluxConfig` and `GitOpsConfig`.
+
+## Flux Configuration
+The flux configuration spec has three optional fields, regardless of the chosen git provider.
+
+### Flux Configuration Spec Details
+### __systemNamespace__ (optional)
+* __Description__: Namespace in which to install the gitops components in your cluster. Defaults to `flux-system`
+* __Type__: string
+
+### __clusterConfigPath__ (optional)
+
+* __Description__: The path relative to the root of the git repository where EKS Anywhere will store the cluster configuration files. Defaults to the cluster name
+* __Type__: string
+
+### __branch__ (optional)
+
+* __Description__: The branch to use when committing the configuration. Defaults to `main`
+* __Type__: string
+
+EKS Anywhere currently supports two git providers for FluxConfig: Github and Git.
+
+## Github provider
```

Comment:
```suggestion
### Github provider
```
I added a # to this heading because I think you mean it to be a subhead of "Flux Configuration".